### DISCUSSION TITLE
Use cowsay in the header

### DISCUSSION THREAD

**Discussion by justinesmithies** [goolord/alpha-nvim#131] [Q&A]
Is it possible to use cowsay in the header just like you can in startify ? If so can you show me an example config for such a setup.

**Comment by geodimm** [comment]
You can probably do something like that which gets you pretty close. You need to have `cowsay` installed on your system though. With a little bit of tweaking it could look even better:

```lua
local cowsay = function()
  local max_width = 80
  local fortune_result = require('alpha.fortune')({ max_width = max_width })
  local text = table.concat({ unpack(fortune_result, 2, #fortune_result) }, '\n')
  local result = vim.fn.system(string.format('cowsay -W %s "%s"', max_width, text))
  return result
end
```

Then it depends on your config, but you can do something along the lines of

```lua
require'alpha.themes.dashboard'.section.header.val = cowsay()
require'alpha'.setup(require'alpha.themes.dashboard'.opts)

```
